Common/python/Methods/method-copy-odoo.sublime-snippet

```diff
@@ -1,7 +1,7 @@
 <!-- Odoo Python Snippets -->
 
 <snippet>
-    <content><![CDATA[@api.multi
+    <content><![CDATA[
     def copy(self, default=None):
```

Comment:
start the `def ` just after the `CDATA[def copy(self, default=None):`

If you don't do this every time you use this snippet you will have an extra identation